samples/bluetooth/broadcaster_multiple/src/broadcaster_multiple.c

```diff
@@ -27,7 +27,11 @@
 #define BT_AD_DATA_FORMAT_LEN_MAX 255U
 
 /* Device name length, size minus one null character */
-#define BT_DEVICE_NAME_LEN (sizeof(CONFIG_BT_DEVICE_NAME) - 1U)
+#if defined(CONFIG_BT_DEVICE_NAME_DYNAMIC)
+#define BT_DEVICE_NAME_LEN (CONFIG_BT_DEVICE_NAME_MAX + 1U)
+#else
+#define BT_DEVICE_NAME_LEN sizeof(CONFIG_BT_DEVICE_NAME)
+#endif
```

Comment:
This change now seems unnecessary

Reply:
It is not really necessary, but is a more generic way of setting the device name. I prefer leaving it this way.